data/fh/scenarios/030.json

```diff
@@ -22,10 +22,12 @@
   "monsters": [
     "algox-archer",
     "algox-guard",
-    "algox-icespeaker",
```

Comment:
This should not be removed! Monsters are still monsters, the `allies` field only declares which of the monsters in list are allies. So Removing this here can break things on certain settings without auto standees etc.